hezronkimutai/politico-api
# [Fixes #163896975] Changes the database format

**What does this PR do?**
This PR changes the the format of the datastructure database from a dictionary in a list to a dictionary in a dictionary
**Description of tasks to**
- Modify the files in the modls modules so that data is stored in a dictionarynrather than a list

**How should this be manually tested?**
- Clone this repo by running the command _git clone https://github.com/hezronkimutai/politico-api.git_
- Install requirements.txt using _pip3 install -r requirements.txt_ 
- On testing the endpoints on postman the results like those provided below should be viewed
**What are the relevant pivotal tracker stories?**

[#163896975](https://www.pivotaltracker.com/n/projects/2243268)

**Screenshots (if appropriate)**
![createoffice](https://user-images.githubusercontent.com/45129725/52627549-b82efc00-2ec6-11e9-81e7-6a5e149fa25a.png)
![create office](https://user-images.githubusercontent.com/45129725/52627551-b8c79280-2ec6-11e9-90ae-c0285050e42e.png)
![createparty](https://user-images.githubusercontent.com/45129725/52627553-b9602900-2ec6-11e9-8225-c2d2cb2a1988.png)
![deleteoffice](https://user-images.githubusercontent.com/45129725/52627554-b9f8bf80-2ec6-11e9-8599-351ec0a6c6f7.png)
![editoffice](https://user-images.githubusercontent.com/45129725/52627555-b9f8bf80-2ec6-11e9-890f-5485563dcca8.png)
![editoffice1](https://user-images.githubusercontent.com/45129725/52627558-ba915600-2ec6-11e9-98b4-8f3aa53307bf.png)
![editreadme](https://user-images.githubusercontent.com/45129725/52627561-ba915600-2ec6-11e9-9ba2-0f40d5bb5d88.png)
![getallparties](https://user-images.githubusercontent.com/45129725/52627562-ba915600-2ec6-11e9-9fb6-42b5fb3b3b61.png)
![get_all users](https://user-images.githubusercontent.com/45129725/52627563-bb29ec80-2ec6-11e9-94f2-7a75dfc406cc.png)
![getdeleteoffce](https://user-images.githubusercontent.com/45129725/52627564-bb29ec80-2ec6-11e9-8cf2-766b38c338de.png)
![getsingleoffice](https://user-images.githubusercontent.com/45129725/52627565-bbc28300-2ec6-11e9-86f5-449857dbcfd1.png)
![get single party](https://user-images.githubusercontent.com/45129725/52627566-bbc28300-2ec6-11e9-8383-d578be230e9f.png)
![get single user](https://user-images.githubusercontent.com/45129725/52627567-bbc28300-2ec6-11e9-8000-c0c031a26ac3.png)
